stories/utility/Grid.stories.js

```diff
@@ -1,100 +1,162 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
-import { action } from '@storybook/addon-actions';
 import StoryContainer from '../storyHelpers/StoryContainer';
-import { Grid } from '@sparkpost/matchbox';
+import { Box, Grid } from '@sparkpost/matchbox';
 
-const DemoBlock = ({size = '18px'}) => {
-  return (
-    <div style={{
-      backgroundColor: '#fa6423',
-      height: size,
-      borderRadius: '2px',
-      marginBottom: '18px'
-    }} />
-  );
-}
+const DemoBlock = ({ height = '18px' }) => {
+  return <Box bg="blue.400" height={height} mb="18px" />;
+};
 
 storiesOf('Utility|Grid', module)
-  .addDecorator((getStory) => (
-    <StoryContainer>{ getStory() }</StoryContainer>
-  ))
-
-  .add('Responsive', withInfo({ propTablesExclude: [DemoBlock] })(() => (
-    <Grid>
-      <Grid.Column xs={12} md={2} lg={6}><DemoBlock /></Grid.Column>
-      <Grid.Column xs={12} md={5} lg={3}><DemoBlock /></Grid.Column>
-      <Grid.Column xs={12} md={5} lg={3}><DemoBlock /></Grid.Column>
-    </Grid>
-  )))
-
-  .add('Auto Width', withInfo({ propTablesExclude: [DemoBlock] })(() => (
-    <Grid>
-      <Grid.Column><DemoBlock /></Grid.Column>
-      <Grid.Column><DemoBlock /></Grid.Column>
-      <Grid.Column><DemoBlock /></Grid.Column>
-      <Grid.Column><DemoBlock /></Grid.Column>
-    </Grid>
-  )))
-
-  .add('Offset', withInfo({ propTablesExclude: [DemoBlock] })(() => (
-    <Grid>
-      <Grid.Column xsOffset={10} xs={2}><DemoBlock /></Grid.Column>
-      <Grid.Column xsOffset={8} xs={4}><DemoBlock /></Grid.Column>
-      <Grid.Column xsOffset={6} xs={6}><DemoBlock /></Grid.Column>
-      <Grid.Column xsOffset={4} xs={8}><DemoBlock /></Grid.Column>
-      <Grid.Column xsOffset={2} xs={10}><DemoBlock /></Grid.Column>
-    </Grid>
-  )))
+  .addDecorator(getStory => <StoryContainer bg="white">{getStory()}</StoryContainer>)
```

Comment:
Sorry, I didnt explain correctly – `storiesOf` is deprecated, I believe. The newer storybook usage uses JS exports to define stories. Take a look at `Button.stories.js` for example